src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,48 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService manufacturerService =
+                (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        DriverService driverService =
+                (DriverService) injector.getInstance(DriverService.class);
+
+        Manufacturer manufacturer = new Manufacturer("Zaz", "Ukraine");
+        System.out.println(manufacturerService.create(manufacturer));
+
+        Driver driver = new Driver("Joey", "КАІ 12345");
+        System.out.println(driverService.create(driver));
+
+        manufacturer.setName("Dacia");
+        manufacturer.setCountry("Romania");
+        manufacturer.setName("VAG");
+        manufacturer.setCountry("Germany");
+        manufacturer.setName("Seat");
+        manufacturer.setCountry("Spain");
```

Comment:
u can create and use constructor. Not critical

Reply:
Please fix all mistakes from previous homework